library/src/interactor_impl.cxx

```diff
@@ -334,6 +358,30 @@ class interactor_impl::internals
   //----------------------------------------------------------------------------
   void StartInteractor()
   {
+#if F3D_MODULE_UI
+    // create event loop to refresh UI and trigger full render if needed
```

Comment:
add a link to issue that this will need rework

Reply:
Will it? What do you have in mind for the rework?

Reply:
Im thinking about it but nothing is clear yet, in any case the issue is this one: https://github.com/f3d-app/f3d/issues/1534